plugin/pkg/scheduler/algorithm/priorities/priorities.go

```diff
@@ -170,6 +170,77 @@ func (n *NodeLabelPrioritizer) CalculateNodeLabelPriority(pod *api.Pod, machines
 	return result, nil
 }
 
+// This is a reasonable size range of all container images. 90%ile of images on dockerhub drops into this range.
+const minImgSize int64 = 23000000   // min 23MB
+const maxImgSize int64 = 1000000000 // max 1000MB
+
+// ImageLocalityPriority is a priority function that favors nodes with requested containers' images of a Pod.
+// It will detect whether the requested images are present on a node, and then calculate a score
+// range from 0 to 10 based on the sum size of those images:
+// - If all the images are not present on a node, this node will be given the lowest priority.
+// - If some of the images are present on a node, the larger their sizes' sum, the higher the node's priority.
+func ImageLocalityPriority(pod *api.Pod, machinesToPods map[string][]*api.Pod, podLister algorithm.PodLister, nodeLister algorithm.NodeLister) (schedulerapi.HostPriorityList, error) {
+	var score int
```

Comment:
Explicitly set the score to 0?
score := 0

It is more obvious that way than relying on Go's zero-values.